{
  "version": 4,
  "valid_for_interpreter_constraints": [
    "CPython==3.11.*"
  ],
  "generated_with_requirements": [
    "PyGithub==2.4.0",
    "PyYAML<7.0,>=6.0",
    "ansicolors==1.1.8",
    "beautifulsoup4==4.11.1",
    "chevron==0.14.0",
    "debugpy==1.8.13",
    "fastapi==0.78.0",
    "fasteners==0.16.3",
    "freezegun==1.2.1",
    "hdrhistogram==0.10.3",
    "ijson==3.2.3",
    "libcst==1.4.0",
    "mypy-typing-asserts==0.1.1",
    "node-semver==0.9.0",
    "packaging==24.2",
    "pex==2.61.1",
    "psutil==5.9.8",
    "pydevd-pycharm==251.23536.40",
    "pytest!=7.1.0,!=7.1.1,<9,>=7",
    "python-gnupg==0.4.9",
    "python-lsp-jsonrpc==1.0.0",
    "requests[security]>=2.28.1",
    "setproctitle==1.3.2",
    "setuptools<64.0,>=63.1.0",
    "starlette==0.19.1",
    "strawberry-graphql[fastapi]==0.114.0",
    "toml==0.10.2",
    "types-PyYAML==6.0.3",
    "types-freezegun==1.1.6",
    "types-requests==2.28.1",
    "types-setuptools==62.6.1",
    "types-toml==0.10.8",
    "typing-extensions~=4.12",
    "urllib3<2",
    "uvicorn[standard]==0.17.6"
  ],
  "manylinux": "manylinux2014",
  "requirement_constraints": [],
  "only_binary": [],
  "no_binary": [],
  "excludes": [],
  "overrides": [],
  "description": "This lockfile was generated by Pants. To regenerate, run: pants generate-lockfiles --resolve=python-default"
}